server/src/concepts/user/controllers/index.js

```diff
@@ -1,15 +1,40 @@
 import validateCreateUser from "../model/UserValidation.js";
 import { createNewUser } from "../useCases/createNewUser.js";
+import { updateUserFunc } from "../useCases/updateUser.js";
+import { getUser } from "../repositories/queries.js";
 
 export const createUser = async (req, res) => {
+  let user;
+
   const validationResult = validateCreateUser(req.body);
   if (validationResult.error) {
-    return res.status(400).send("Invalid credentials");
+    return res.status(400).send("Invalid data");
+  }
+
+  try {
+    user = await createNewUser(req.body);
```

Comment:
```suggestion
    const user = await createNewUser(req.body);
      return res.status(200).send({
    message: "User was added",
    data: user,
  });
```

We can immediately return here in try block instead of doing it below. In my opinion it is better approach